metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi.py

```diff
@@ -827,7 +808,7 @@ def generate_container_for_workspace(
         container_work_units = gen_containers(
             container_key=self.workspace_key,
             name=workspace.name,
-            sub_types=[BIContainerSubTypes.POWERBI_WORKSPACE],
+            sub_types=[workspace.type],
```

Comment:
is BIContainerSubTypes.POWERBI_WORKSPACE used anywhere now?

Reply:
removed